assets/javascripts/kitten/components/information/alert-box/index.js

```diff
@@ -173,10 +178,10 @@ export const AlertBox = ({
 
   const iconSize = (() => {
     switch(size) {
-      case 'big':
-        return 'normal'
-      case 'normal':
-        return 'tiny'
+      case 'large' || 'big':
+        return 'medium' || 'normal'
+      case 'medium' || 'normal':
+        return 'small' || 'tiny'
```

Comment:
Normalement ça marche aussi comme ça

```suggestion
      case 'large':
      case 'big':
        return 'medium'
      case 'medium':
      case 'normal':
        return 'small'
```